requirements.in

```diff
@@ -1,4 +1,4 @@
-hca-ingest~=2.5.1
+hca-ingest~=2.5.2
```

Comment:
we really don't need the version number here, since the specific version is listed in the requirements.txt